### PR TITLE
docs: Add Code of Conduct

### PR DESCRIPTION
**Pull request by avaly**
To attract more open source contributors we should establish some basic guidelines of working together.

I've used the Code of Conduct document found at https://www.contributor-covenant.org/, which seems to be widely used by a number of big open source projects.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+report@plexapp.com.
```
Comment (avaly) [inline CODE_OF_CONDUCT.md:63]: I made this up. Do we have a generic email we could use here?

Comment (ejmartin504) [reply CODE_OF_CONDUCT.md:63]: avaly@plexapp.com

**Review by ejmartin504 — APPROVED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (sullman) [reply CODE_OF_CONDUCT.md:63]: Hmm, not that I know of, we might need to create one.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+- Focusing on what is best not just for us as individuals, but for the overall
+  community
+
+Examples of unacceptable behavior include:
+
+- The use of sexualized language or imagery, and sexual attention or advances of
+  any kind
+- Trolling, insulting or derogatory comments, and personal or political attacks
+- Public or private harassment
+- Publishing others' private information, such as a physical or email address,
```
Comment (sullman) [inline CODE_OF_CONDUCT.md:34]: Did Teddy already break this rule? :stuck_out_tongue:

**Review by sullman — DISMISSED** [review]

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (ejmartin504) [reply CODE_OF_CONDUCT.md:34]: :hide:

**Review by sullman — APPROVED** [review]

**Pull request merged**